### PR TITLE
LP-37 implement flyway configuration

### PR DESCRIPTION
**Pull request by brunaellen**
* LP-37 - Adding flyway configuration
* Testing the Flyway configuration
* Tidying up the dependencies.

### CODE REVIEW

File: src/main/resources/db/migration/V1__Create_table.sql
Comment (joycesilvamartins) [inline src/main/resources/db/migration/V1__Create_table.sql:1]: As we discussed in the meeting. Maybe we can remove this file.

File: src/main/resources/db/migration/V1__Create_table.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS `employee` (
```
Comment (joycesilvamartins) [inline src/main/resources/db/migration/V1__Create_table.sql]: As we discussed in the meeting. Maybe we can remove this file.

File: src/test/java/com/java/study/group/librarysystem/db/FlywayTest.java
Comment (joycesilvamartins) [inline src/test/java/com/java/study/group/librarysystem/db/FlywayTest.java:1]: As we discussed in the meeting. Maybe we can use the Java NIO (New Input/Output) API to make this test dynamic.

**Review by joycesilvamartins — APPROVED** [review]

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -14,7 +16,5 @@ spring.datasource.password=
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
 spring.jpa.hibernate.ddl-auto=update
 spring.jpa.defer-datasource-initialization=true
-spring.jpa.properties.hibernate.show_sql=true
```
Comment (RaquelOliveira) [inline src/main/resources/application.properties:17]: Maybe we don't need to remove this property from H2. YDYT?

**Review by RaquelOliveira — APPROVED** [review]
Thank you for putting so much effort into this.

**Pull request merged**